Update incompatible-plugins.md

TT-Candy opened the pull request:
更新不兼容配置页面 加入了一些常见的插件兼容问题

TT-Candy committed:
更新中文配置文件

TT-Candy committed:
脚本真方便

TT-Candy committed:
一阶段更新

TT-Candy committed:
二次更新完成 希望没问题了

TT-Candy committed:
漏东西了

TT-Candy committed:
一个bug 一个遗漏

TT-Candy committed:
完善两个描述

TT-Candy committed:
这应该是最后的修改了

TT-Candy committed:
又漏东西了 这回真的是最后的修改了

TT-Candy committed:
今天凌晨的更新 这回真的真的是最后的修改了

TT-Candy committed:
Merge branch 'LeavesMC:main' into main

TT-Candy committed:
Merge branch 'LeavesMC:main' into main

TT-Candy committed:
Merge branch 'LeavesMC:main' into main

TT-Candy committed:
Merge branch 'LeavesMC:main' into main

TT-Candy committed:
更新不兼容插件说明和建议

更新不兼容插件的描述，增强了对特定插件的兼容性说明，并提供了替代插件的建议。

LittleChest commented on src/zh_Hans/leaves/reference/incompatible-plugins.md:
```diff
@@ -1,37 +1,51 @@
 # 不兼容的插件
 
-Leaves 尽力与所有 CraftBukkit、Spigot 或 Paper 插件兼容，但一些插件需要额外的配置甚至使用替代品才能与 Leaves 协同工作。
+Leaves 核心致力于与所有 CraftBukkit、Spigot 及 Paper 插件保持兼容，但部分插件因技术或功能的冲突，可能需要额外配置或采用替代方案以实现兼容运行。
```
避免提及“核心”

LittleChest commented on src/zh_Hans/leaves/reference/incompatible-plugins.md:
```diff
@@ -1,37 +1,51 @@
+Leaves 核心致力于与所有 CraftBukkit、Spigot 及 Paper 插件保持兼容，但部分插件因技术或功能的冲突，可能需要额外配置或采用替代方案以实现兼容运行。
 
-**我们不会处理与这些插件相关的问题，请在报告问题前暂时移除它们。**
+**对于此类插件引发的兼容性问题，我们无法提供支持服务。如需提交问题反馈，请先暂时移除相关插件以排除干扰。**
```
没有原句子通顺（？

原句子应该加这个
```suggestion
**我们不会处理与这些插件相关的问题，请在报告问题前暂时移除它们并尝试复现问题。**
```

LittleChest commented on src/zh_Hans/leaves/reference/incompatible-plugins.md:
```diff
@@ -1,37 +1,51 @@
+## 数据转译类插件
 
-这些插件会破坏 Leaves 的协议支持，导致模组无法正常通讯或玩家被踢出。
+Leaves 核心的模组协议功能依赖服务端与客户端之间的准确数据通信，使用涉及数据转译的插件（如 ViaVersion、ViaBackwards、ViaRewind、Geyser）以及代理服务端（如 Velocity、BungeeCord、ViaProxy ）可能造成客户端模组功能异常或玩家无法连接至服务器。
```
同上 避免提及“核心”
副标题添加“插件”有点多余 这篇文章说的就是插件 ~~有人会联想到别的吗~~
Geyser不属于数据转译范畴，它完完全全模拟一个原版Java版客户端，理论上不应存在问题
Velocity完全转发客户端数据 理论上不应存在问题 值得一提的是 Velocity上也可以安装Via

LittleChest commented on src/zh_Hans/leaves/reference/incompatible-plugins.md:
```diff
@@ -1,37 +1,51 @@
-### ViaVersion
+::: tip
+
+对于跨版本需求，请优先考虑在客户端安装[ViaFabricPlus](https://github.com/ViaVersion/ViaVersion)模组**并手动设置连接至服务器所使用的版本。**
```
添加ViaVanillaPlus

LittleChest commented on src/zh_Hans/leaves/reference/incompatible-plugins.md:
```diff
@@ -1,37 +1,51 @@
+::: tip
+
+对于跨版本需求，请优先考虑在客户端安装[ViaFabricPlus](https://github.com/ViaVersion/ViaVersion)模组**并手动设置连接至服务器所使用的版本。**
+若必须使用上述插件或代理端，请在必要时禁用**所有**模组协议功能。
```
同上

LittleChest commented on src/zh_Hans/leaves/reference/incompatible-plugins.md:
```diff
@@ -1,37 +1,51 @@
+
+:::
+
+## 登录插件
```
同上

LittleChest commented on src/zh_Hans/leaves/reference/incompatible-plugins.md:
```diff
@@ -1,37 +1,51 @@
+## 登录插件
 
-ViaVersion 只翻译原版协议，而不会处理模组协议。这会导致模组协议版本跨度过大无法正确解析。
+常规登录插件（如 AuthMe、AuthMeReloaded、CatSeedLogin）无法正确识别假人，并错误地要求假人进行登录，这将导致假人无法生成或功能执行异常。
```
“常规登录插件” 有严重的偏见色彩 不建议使用

LittleChest commented on src/zh_Hans/leaves/reference/incompatible-plugins.md:
```diff
@@ -1,37 +1,51 @@
 ::: tip
 
-客户端可以安装 [ViaFabricPlus](https://modrinth.com/mod/viafabricplus) 和 [ViaVanillaPlus](https://modrinth.com/mod/viavanillaplus) 以翻译部分模组协议。
+建议使用[AuthMeReReloaded](https://github.com/HaHaWTH/AuthMeReReloaded)登录插件，此插件能正确识别假人并允许其自动绕过登录流程。
```
中英文之间使用空格
不应直接推荐插件 ~~这样像是广告~~ 应注明其是AuthMe的Fork

LittleChest commented on src/zh_Hans/leaves/reference/incompatible-plugins.md:
```diff
@@ -1,37 +1,51 @@
 
-客户端可以安装 [ViaFabricPlus](https://modrinth.com/mod/viafabricplus) 和 [ViaVanillaPlus](https://modrinth.com/mod/viavanillaplus) 以翻译部分模组协议。
+建议使用[AuthMeReReloaded](https://github.com/HaHaWTH/AuthMeReReloaded)登录插件，此插件能正确识别假人并允许其自动绕过登录流程。
+若无法更换登录插件，可安装[LeavesAddons](https://github.com/Lumine1909/LeavesAddons)插件以实现兼容。
```
同上
移除前半句

LittleChest commented on src/zh_Hans/leaves/reference/incompatible-plugins.md:
```diff
@@ -1,37 +1,51 @@
 :::
 
-## 行为
+## 领地插件
```
同上

LittleChest commented on src/zh_Hans/leaves/reference/incompatible-plugins.md:
```diff
@@ -1,37 +1,51 @@
+## 多世界插件
 
-AuthMeReloaded 不会正确检测假人并错误的要求假人登录，导致假人无法正确生成。
+在部署多世界插件（如 Multiverse）时，若配置不当可能导致部分实体无法通过传送门进入指定维度，或出现传送功能完全失效的情况。需特别指出，重力方块的复制机制仅适用于末地维度，请务必确保末地传送门正确链接到末地维度。
```
这和Leaves没关系
本篇文章描述的是插件破坏Leaves功能的情况

换句话说 这插件扔Paper上也会炸

LittleChest commented on src/zh_Hans/leaves/reference/incompatible-plugins.md:
```diff
@@ -1,37 +1,51 @@
-:::
+## 优化插件
+
+很多红石机器的运作高度依赖游戏的特性和底层机制，而大多数性能优化插件会对原版机制进行不同程度的修改。这些修改在常规游戏场景中通常不会被感知，但在精密的红石机器中，微小的机制偏差将被显著放大，最终导致机器无法正常工作。
```
这和Leaves没关系
本篇文章描述的是插件破坏Leaves功能的情况

换句话说 这插件扔Paper上也会炸

LittleChest commented on src/zh_Hans/leaves/reference/incompatible-plugins.md:
```diff
@@ -1,37 +1,51 @@
+
+### Aki-Async
+
+异步优化通过多线程计算提升服务端性能，但此举必然干扰游戏原生时序逻辑，破坏微时序机制，并引入线程安全隐患和崩溃风险。这将破坏原版特性，并引发功能异常问题，极端情况下甚至造成服务端崩溃。此外，多线程计算非常复杂，错误的优化不仅无法实现预期效果，反而会导致服务器性能损耗加剧，引发卡顿。
```
这和Leaves没关系
本篇文章描述的是插件破坏Leaves功能的情况

换句话说 这插件扔Paper上也会炸

TT-Candy replied:
这里推荐保留 因为这是一个基于Leaves Mixin制作的插件 或者说它是一个Leaves插件

LittleChest replied:
Leaves插件不兼容Leaves吗（

LittleChest commented on src/zh_Hans/leaves/reference/incompatible-plugins.md:
```diff
@@ -1,37 +1,51 @@
 ### NoCheatPlus
 
-NoCheatPlus 会覆盖 Paper 的可配置修复项，导致部分红石机器无法使用。
+NoCheatPlus 插件会强制修改 Paper 核心的部分配置项，导致部分原版特性（如活塞复制、无头活塞，重力方块复制）缺失。
```
强制修改 -> 覆盖
修改指将变更写入到磁盘

避免提及核心

不应使用缺失 而是它们被作为漏洞修复了
可以强调即使启用生电模式 这些配置项也不会被Leaves覆盖

TT-Candy committed:
Revise compatibility information for plugins

Updated compatibility notes for various plugins, including data translation, login, territory, multi-world, and optimization plugins. Clarified issues and provided alternative solutions.

LittleChest commented on src/zh_Hans/leaves/reference/incompatible-plugins.md:
```diff
@@ -1,37 +1,47 @@
 ::: tip
 
-客户端可以安装 [ViaFabricPlus](https://modrinth.com/mod/viafabricplus) 和 [ViaVanillaPlus](https://modrinth.com/mod/viavanillaplus) 以翻译部分模组协议。
+对于跨版本需求，请优先考虑在客户端安装 [ViaFabricPlus](https://github.com/ViaVersion/ViaFabricPlus) 和 [ViaVanillaPlus](https://github.com/ViaVersionAddons/ViaVanillaPlus) 模组**并手动设置连接至服务器所使用的版本。**
```
句号不应被加粗

LittleChest commented on src/zh_Hans/leaves/reference/incompatible-plugins.md:
```diff
@@ -1,37 +1,47 @@
+可以尝试以下解决方案：
 
-::: tip
+1.使用 [AuthMeReReloaded](https://github.com/HaHaWTH/AuthMeReReloaded) 登录插件，此插件为 AuthMeReloaded 插件的分支，它能正确识别假人并允许其自动绕过登录流程。
```
序号后添加空格 就像这样
`1. 一`

LittleChest commented on src/zh_Hans/leaves/reference/incompatible-plugins.md:
```diff
@@ -1,37 +1,47 @@
-::: tip
+1.使用 [AuthMeReReloaded](https://github.com/HaHaWTH/AuthMeReReloaded) 登录插件，此插件为 AuthMeReloaded 插件的分支，它能正确识别假人并允许其自动绕过登录流程。
 
-请使用替代品 [AuthMeReReloaded](https://hangar.papermc.io/0D00_0721/AuthMeReReloaded)，它添加了对假人的支持，并且与 AuthMeReloaded 的使用近乎相同。
```
对于已上架到Hanger/Modrinth的插件 应优先使用这里的链接 对用户友好

LittleChest commented on src/zh_Hans/leaves/reference/incompatible-plugins.md:
```diff
@@ -1,37 +1,47 @@
+## NoCheatPlus
 
-NoCheatPlus 会覆盖 Paper 的可配置修复项，导致部分红石机器无法使用。
+NoCheatPlus 插件会覆盖 Paper 的部分配置项，即便启用了 Leaves 的 `mc-technical-survival-mode` 配置也无法将其恢复。这将导致部分原版特性（如活塞复制、无头活塞，重力方块复制）被修复。
```
最后一句话有点重复
```suggestion
NoCheatPlus 插件会覆盖 Paper 的部分修复项（如活塞复制、无头活塞、重力方块复制），即便启用了 Leaves 的 `mc-technical-survival-mode` 配置也无法将其恢复。
```

TT-Candy committed:
Update plugin links and clarify NoCheatPlus effects

TT-Candy committed:
Remove Aki-Async plugin warnings from documentation

Removed section on Aki-Async plugin and its potential issues.

LittleChest commented on src/zh_Hans/leaves/reference/incompatible-plugins.md:
```diff
@@ -1,37 +1,43 @@
+::: tip
 
-AuthMeReloaded 不会正确检测假人并错误的要求假人登录，导致假人无法正确生成。
+可以尝试以下解决方案：
```
描述一下 这些解决方案是给AuthMe用的

```suggestion
当使用 AuthMeReload 时可以尝试以下解决方案：
```

LittleChest commented on src/zh_Hans/leaves/reference/incompatible-plugins.md:
```diff
@@ -1,37 +1,43 @@
 # 不兼容的插件
 
-Leaves 尽力与所有 CraftBukkit、Spigot 或 Paper 插件兼容，但一些插件需要额外的配置甚至使用替代品才能与 Leaves 协同工作。
+Leaves 致力于与所有 CraftBukkit、Spigot 及 Paper 插件保持兼容，但部分插件因技术或功能的冲突，可能需要额外配置或采用替代方案以实现兼容运行。
```
```suggestion
Leaves 致力于与所有 CraftBukkit、Spigot 及 Paper 插件保持兼容，但部分插件可能需要额外配置或采用替代方案以正常运行。
```

TT-Candy committed:
Revise incompatible plugins documentation

Updated compatibility information for plugins, including additional details for AuthMeReReloaded and LeavesAddons.

LittleChest commented on src/zh_Hans/leaves/reference/incompatible-plugins.md:
```diff
@@ -24,9 +24,9 @@ Leaves 的模组协议功能依赖服务端与客户端之间的准确数据通
+1. 使用 [AuthMeReReloaded](https://hangar.papermc.io/0D00_0721/AuthMeReReloaded) 登录插件，此插件为 AuthMeReloaded 插件的分支，它能正确识别假人并允许其自动绕过登录流程，并支持从其他登录插件迁移数据。
 
-2. 安装 [LeavesAddons](https://github.com/Lumine1909/LeavesAddons) 插件以实现兼容。
+2. 对于 Authme 插件，可以安装 [LeavesAddons](https://github.com/Lumine1909/LeavesAddons) 插件以实现兼容。
```
大小写 而且现在哪有AuthMe 不是AuthMeReload吗（
